sdcm/scylla_bench_thread.py

```diff
@@ -146,31 +146,33 @@ def _run_stress_bench(self, node, loader_idx, stress_cmd, node_list):
                                        stress_operation=stress_cmd_opt,
                                        stress_log_filename=log_file_name,
                                        loader_idx=loader_idx), \
-                ScyllaBenchStressEventsPublisher(node=node, sb_log_filename=log_file_name):
+                ScyllaBenchStressEventsPublisher(node=node, sb_log_filename=log_file_name), \
+                ScyllaBenchEvent(node=node, stress_cmd=self.stress_cmd,
```

Comment:
Let's move it out from `with` statement and do finish_event().publish() at get_stress_results_bench

Reply:
fixed